examples/inference.ipynb

```diff
@@ -1879,6 +1901,262 @@
+              "Downloading pytorch_model.bin:   0%|          | 0.00/539M [00:00<?, ?B/s]"
+            ],
+            "application/vnd.jupyter.widget-view+json": {
+              "version_major": 2,
+              "version_minor": 0,
+              "model_id": "71f572d81aad4ad68d84aea524b22ea3"
+            }
+          },
+          "metadata": {}
+        },
+        {
+          "output_type": "display_data",
+          "data": {
+            "text/plain": [
+              "Downloading (…)neration_config.json:   0%|          | 0.00/69.0 [00:00<?, ?B/s]"
+            ],
+            "application/vnd.jupyter.widget-view+json": {
+              "version_major": 2,
+              "version_minor": 0,
+              "model_id": "71da18a1f3104c0baf7e6452499ef33c"
+            }
+          },
+          "metadata": {}
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "A midi input prompt and a [soundfont](https://web.archive.org/web/20130616094125/http://www.geocities.jp/shansoundfont/) (used to render the generated MIDI symbols into WAV format) are provided for the examples in this notebook and can be downloaded using the following cells."
+      ],
+      "metadata": {
+        "id": "xW2xWxUGVhIh"
+      },
+      "id": "xW2xWxUGVhIh"
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "res = requests.get(\"https://martin-krasser.com/perceiver/data/midi/prompt.mid\")\n",
+        "with open(\"prompt.mid\", \"wb\") as f:\n",
+        "    f.write(res.content)"
+      ],
+      "metadata": {
+        "id": "d7V2dzL-Vsre"
+      },
+      "id": "d7V2dzL-Vsre",
+      "execution_count": 43,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "res = requests.get(\"https://martin-krasser.com/perceiver/data/midi/SGM-v2.01-NicePianosGuitarsBass-V1.2.sf2\")\n",
+        "with open(\"SGM-v2.01-NicePianosGuitarsBass-V1.2.sf2\", \"wb\") as f:\n",
+        "    f.write(res.content)"
+      ],
+      "metadata": {
+        "id": "5iW1UvFDVs5d"
+      },
+      "id": "5iW1UvFDVs5d",
+      "execution_count": 44,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "prompt = PrettyMIDI(\"prompt.mid\")"
+      ],
+      "metadata": {
+        "id": "Idbid6snVzQo"
+      },
+      "id": "Idbid6snVzQo",
+      "execution_count": 45,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "The following example demonstrates how to create a continuation of an input prompt. It uses `512` tokens from the input prompt to generate `1024` new tokens. The initial number of latent positions is set to `1` corresponding to the last `1` tokens of the prompt sequence. All other prompt tokens are used as prefix tokens. The generated sequence is appended to the input prompt (`return_full_audio=True`) and the output is return as a binary wav sequence. The generated sequence starts at timestamp `14s`."
```

Comment:
output is return<ins>ed</ins> ~~as a binary wav sequence~~ in binary wav format.